pkg/gui.py

```
@@ -348,3 +332,89 @@ class SettingsMenu(OtherFrame):
+
+        self.update_settings(settings)
+
+    def save_settings(self):
+        # Get the form values
+        tesseract_path = self.tesseract_path_entry.get()
+        debug_level = self.debug_level_var.get()
+
+        # Create a dictionary to hold the settings
+        settings = {
+            "tesseract_path": tesseract_path,
+            "debug_level": debug_level
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)